### PR TITLE
fix: #6182 - make favicon visible in dark-themed browser.

### LINKED ISSUE #6182: favicon is invisible in the dark-themed browser
image

### PR DESCRIPTION
**Pull request by shane215**
Fixes #6182.

I tested by following the approach outlined [here](https://community.jitsi.org/t/how-to-test-changes-to-index-html-on-localhost/110831/2?u=shane.215), and also by turning on dark mode for web browser and Windows OS.

Results:

- Chrome (new on left, current on right)
  - dark-mode-chrome_compareToCurrentIcon

- Firefox (new on left, current on right)
  - dark-mode-firefox_compareToCurrentIcon

- Edge (new on left, current on right)
  - dark-mode-edge_compareToCurrentIcon

- Safari (not tested)
  - I have Windows so I cannot test on Safari.
  - This [page](https://support.apple.com/en-us/HT204416) confirms that Apple no longer offers Safari updates for Windows. Safari 5.1.7 for Windows was the last version made for Windows, and it is now outdated.
  - Also, Apple's website only offers the latest version of Safari, so I can't even get 5.1.7.

### CODE REVIEW

**Comment by saghul** [comment]
Awesome! Should the current favicon go away now?

**Comment by shane215** [comment]
> Awesome! Should the current favicon go away 

I think that only one favicon file should be needed and that’s now added in index.html.

However, there’s at least 2 existing files named “favicon”:

1. [/favicon.ico](https://github.com/jitsi/jitsi-meet/blob/master/favicon.ico)
1. [images/favicon.ico](https://github.com/jitsi/jitsi-meet/blob/master/images/favicon.ico)

I see ‘images/favicon.ico’ used in [title.html](https://github.com/jitsi/jitsi-meet/blob/master/title.html).

I don’t know how exactly these other favicons are used, perhaps they should be removed.

Can someone confirm this?

File: index.html [hunk 1]
```diff
@@ -7,6 +7,7 @@
     <meta name="theme-color" content="#2A3A4B">
     <!--#include virtual="base.html" -->
 
+    <link rel="icon" href="images/favicon.svg">
```
Comment (saghul) [inline index.html:10]: this should go on title.html, also please update the Makefile to drop the reference to the old file.

**Review by saghul — CHANGES_REQUESTED** [review]
Yes please let's drop the old one. Have you tested Safari too?

File: index.html (hunk 1 above)
Comment (shane215) [reply index.html:10]: Ok, I'll work on this and also test for Safari.

Comment (shane215) [reply index.html:10]: @saghul 

About testing on Safari, I have Windows so I cannot test on Safari.

This [page](https://support.apple.com/en-us/HT204416) confirms that _Apple no longer offers Safari updates for Windows. Safari 5.1.7 for Windows was the last version made for Windows, and it is now outdated._

Also, Apple's website only offers the latest version of Safari, so I can't even get 5.1.7.

Comment (shane215) [reply index.html]: @saghul I made the other changes that you suggested.

File: title.html [hunk 2]
```diff
@@ -6,4 +6,4 @@
 <meta itemprop="name" content="Jitsi Meet"/>
 <meta itemprop="description" content="Join a WebRTC video conference powered by the Jitsi Videobridge"/>
 <meta itemprop="image" content="images/jitsilogo.png?v=1"/>
-<link rel="icon" type="image/png" href="images/favicon.ico?v=1"/>
+<link rel="icon" href="images/favicon.svg">
```
Comment (saghul) [inline title.html:9]: Can you pl add the `?v=1` ? It will help with caching.

**Review by saghul — COMMENTED** [review]
LGTM with a nit, thank you!

File: title.html (hunk 2 above)
Comment (shane215) [reply title.html]: Ok, I added it.

**Review by saghul — APPROVED** [review]
LGTM! @horymury can you PTAL too, and check the branding implications for 8x8 deployments? Thank you! 🙏

**Comment by saghul** [comment]
Damn this fell through the cracks :-/ @horymury can you PTAL?

**Comment by horymury** [comment]
> LGTM! @horymury can you PTAL too, and check the branding implications for 8x8 deployments? Thank you! 🙏

LGTM too, does not affect the 8x8 deployment. Sorry I missed this one.

**Comment by saghul** [comment]
Can you pl take it over and land it?

**Review by horymury — APPROVED** [review]

**Review by horymury — APPROVED** [review]

**Pull request merged**